docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -347,19 +347,52 @@ Explanation:
 
 `+any` changes an earlier proposal where the `FOLLOWERS` count could also be specified. This has been removed to reduce the risk of discrepancies and misconfiguration. See also "Policy Validation" below.
 
+#### Built-in Placement Policies
+
+By default every system will have two placement policies, which can be modified via `ALTER PLACEMENT POLICY` but never dropped:
+
+* `default`: This policy is used only in the event that a policy has not been specified.
+* `system`: This policy is used for internal TiDB system tables.
+
+Some common applications might be to increase the replica count on system or default tables. It is not typically recommended to add constraints to these policies as it will lead to cluster inbalance, but it is possible.
+
+#### Schedule Property
+
+When using either the syntactic sugar or list format for placement rules, PD is free to schedule followers/leaders/voters wherever it decides. For example:
+
+```sql
+CREATE PLACEMENT POLICY `standardplacement1` PRIMARY REGION="us-east-1" REGIONS="us-east-1,us-east-2" FOLLOWERS=4;
+CREATE PLACEMENT POLICY `standardplacement2` LEADER_CONSTRAINTS="[+region=us-east-1]"  FOLLOWERS_CONSTRAINTS="[+region=us-east-1,+region=us-east-2]" FOLLOWERS=4;
+```
+
+- Are each of the followers split equally in us-east-1 and us-east-2?
+- Could the majority of followers be placed in us-east-1? That would ensure fast quorum but reduced fault tolerance.
+
+To address the ambiguity the concept of a "schedule" is introduced, which is the name for a strategy that PD uses to determine where to place instances. For an example:
+
+```sql
+CREATE PLACEMENT POLICY `standardplacement1` PRIMARY REGION="us-east-1" REGIONS="us-east-1,us-east-2" FOLLOWERS=4 SCHEDULE=SURVIVE_REGION_FAILURE();
+```
+
+The following `SCHEDULE` options are available:
+
+* `BALANCED()`: This is the default schedule. Followers will be balanced on leader balance per store. So if there are 15 stores in `us-east-1` and 5 stores in `us-east-2`, you can expect that approximately 3:1 followers will be placed in `us-east-1` to `us-east-2`.
```

Comment:
I think this behavior is hard to clear, actually, the default scheduling does not consider this way, it's more priority to survive. I think we can't use store numbers to explain the `SCHEDULE` behavior because it can change dynamically and may cause some misunderstandings. we can say the replicas' placement behavior,  such as `SURVIVE_REGION_FAILURE()` is expected each region has a replica is more clear. BTW, I don’t think the "BALANCE" policy is needed yet, we recommend deploy with the same nodes.